README.md

```diff
@@ -62,6 +62,8 @@ OPTIONS:
   -r, --right-file <right-file>
                           The right eye media file to merge.
   -q, --quality <quality> Output video quality [0-100]. 50 is a good default value.
+  -c, --color-depth <color-depth>
```

Comment:
I think this would make more sense as a boolean parameter, something like "--force-8-bit-color".

Reply:
Now that you mention it, a boolean makes much more sense.  I think the force is just a bit misleading though.  I believe without that parameter it will ALWAYS be 10bit.  As far as I could tell, it is not trying to detect 8/10 bit.  Is starting with parameter with a number bad practice or even allowed?  Can you think of a better name?

Reply:
By convention, there's only a few verbs that are used with boolean flags. Perhaps the better name would be `--disable-10-bit-color`?

Reply:
I like it! Sorry I didn't have enough experience to think about the Boolean flag and naming convention. All of the 3D Blu-ray sources I have encountered are 8 bit. Probably due to the standards being older.